app/src/main/java/com/kwartracker/android/settings/changePassword/ui/ChangePasswordFragment.kt

```diff
@@ -0,0 +1,20 @@
+package com.kwartracker.android.settings.changePassword.ui
```

Comment:
@MarvinBlanca 
Change package path to `app/src/main/java/com/kwartracker/android/settings/ui/changepassword/ChangePasswordFragment.kt`